
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:3192772) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_4',
  containerIp: '10.0.3.43',
  mitmPort: 1003,
  autoAccessFixed: 1
}
2022-10-23 02:17:10.164 - [Info] MITM Version: 2
2022-10-23 02:17:10.165 - [Info] Auto Access Enabled: true
2022-10-23 02:17:10.165 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-23 02:17:10.172 - [Info] SSH man-in-the-middle server for 10.0.3.43 listening on 10.0.3.1:1003
2022-10-23 02:30:21.038 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:21.045 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:21.067 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:21.067 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:21.068 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:28.045 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:28.046 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:28.047 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:28.047 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:28.048 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:32.268 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:32.269 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:32.270 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:32.270 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:32.271 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:32.274 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:32.274 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:30:48.349 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'none'
2022-10-23 02:30:48.350 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 02:30:57.051 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'none'
2022-10-23 02:30:57.051 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 02:30:57.153 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'none'
2022-10-23 02:30:57.153 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 02:30:57.188 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'none'
2022-10-23 02:30:57.188 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 02:30:57.189 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'none'
2022-10-23 02:30:57.189 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 02:30:57.210 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'none'
2022-10-23 02:30:57.210 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 02:30:57.234 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'none'
2022-10-23 02:30:57.234 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 02:30:58.021 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'none'
2022-10-23 02:30:58.022 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 02:30:58.485 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'password'
2022-10-23 02:30:58.485 - [Debug] [Auto Access] Attacker: 34.107.118.221, Threshold: 1, Attempts: 1
2022-10-23 02:30:58.485 - [Debug] [Auto Access] Compromising the honeypot
2022-10-23 02:30:58.485 - [Debug] [Auto Access] Adding the following credentials: 'service:service'
2022-10-23 02:30:58.601 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-23 02:30:58.603 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-23 02:30:58.604 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.43
2022-10-23 02:30:58.649 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2022-10-23 02:31:00.901 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'none'
2022-10-23 02:31:00.902 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 02:31:02.356 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:31:02.368 - [Debug] [Connection] Attacker connected: 34.107.118.221 | Client Identification: SSH-2.0-Go
2022-10-23 02:31:05.524 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'password'
2022-10-23 02:31:05.524 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2022-10-23 02:31:05.525 - [Debug] [Auth] Attacker: 34.107.118.221 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 02:31:05.570 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'password'
2022-10-23 02:31:05.570 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2022-10-23 02:31:05.571 - [Debug] [Auth] Attacker: 34.107.118.221 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 02:31:05.607 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'password'
2022-10-23 02:31:05.608 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2022-10-23 02:31:05.608 - [Debug] [Auth] Attacker: 34.107.118.221 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 02:31:05.626 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'password'
2022-10-23 02:31:05.627 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2022-10-23 02:31:05.627 - [Debug] [Auth] Attacker: 34.107.118.221 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 02:31:05.627 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'password'
2022-10-23 02:31:05.628 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2022-10-23 02:31:05.628 - [Debug] [Auth] Attacker: 34.107.118.221 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 02:31:05.667 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'password'
2022-10-23 02:31:05.667 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Container user is disabled
2022-10-23 02:31:05.667 - [Debug] [Auth] Attacker: 34.107.118.221 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 02:31:05.929 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'password'
2022-10-23 02:31:05.929 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Container user is disabled
2022-10-23 02:31:05.930 - [Debug] [Auth] Attacker: 34.107.118.221 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 02:31:07.158 - [Debug] [Auth] Attacker 34.107.118.221 trying to authenticate with 'password'
2022-10-23 02:31:07.159 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2022-10-23 02:31:07.159 - [Debug] [Auth] Attacker: 34.107.118.221 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 02:31:10.486 - [Debug] [Connection] Attacker closed the connection
2022-10-23 02:31:10.533 - [Debug] [Connection] Attacker closed the connection
2022-10-23 02:31:10.549 - [Debug] [Connection] Attacker closed the connection
2022-10-23 02:31:10.558 - [Debug] [Connection] Attacker closed the connection
2022-10-23 02:31:10.558 - [Debug] [Connection] Attacker closed the connection
2022-10-23 02:31:10.580 - [Debug] [Connection] Attacker closed the connection
2022-10-23 02:31:10.814 - [Debug] [Connection] Attacker closed the connection
